connect/connect.md

```diff
@@ -0,0 +1,5 @@
+# ClickView Connect API
```

Comment:
The link to this file is missing from the main readme

Reply:
Added